src/SerializerBuilder.php

```diff
@@ -96,18 +102,25 @@ class SerializerBuilder
      */
     private $accessorStrategy;
 
-    public static function create()
+    public static function create(...$args)
     {
-        return new static();
+        return new static(...$args);
     }
 
-    public function __construct()
+    public function __construct(HandlerRegistryInterface $handlerRegistry = null, EventDispatcherInterface $eventDispatcher = null)
     {
         $this->typeParser = new TypeParser();
-        $this->handlerRegistry = new HandlerRegistry();
-        $this->eventDispatcher = new EventDispatcher();
+        $this->handlerRegistry = $handlerRegistry ?: new HandlerRegistry();
```

Comment:
should be ??

Reply:
should be no difference, since can be only `object|null`, right?